### PR TITLE
Added more data to Device struct, inclu…

### PR DESCRIPTION
**Pull request by AaronH88**
…ding an ImageID.

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/models/devices.go [hunk 1]
```diff
@@ -43,4 +43,7 @@ type Device struct {
 	LastSeen      EdgeAPITime `json:"LastSeen"`
 	CurrentHash   string      `json:"CurrentHash,omitempty"`
 	Account       string      `json:"Account"`
+	DeviceName    string      `json:"DeviceName"`
```
Comment (acosferreira) [inline pkg/models/devices.go:46]: why we need the DeviceName if we already have the name?

Comment (AaronH88) [reply pkg/models/devices.go]: Good spot, we dont need it. removing it now

**Comment by acosferreira** [comment]
1. Should we use the currentHash to store the rpm_ostree_deployments.checksum or we'll go directly create the image relationship?
2. Will we use the booted to store which info? on inventory level this indicates the which image is applyed, in our scenario shouldn't be always true? 
3. On UpdateAvailable, should this be filled up by assync process?

**Comment by AaronH88** [comment]
> 1. Should we use the currentHash to store the rpm_ostree_deployments.checksum or we'll go directly create the image relationship?
> 
>     2. Will we use the booted to store which info? on inventory level this indicates the which image is applyed, in our scenario shouldn't be always true?
> 
>     3. On UpdateAvailable, should this be filled up by assync process?

1. I never looked at that relationship yet. Happy to take direction from you if you think either is a good idea :smile: 

2. This was copy and pasted from the `EdgeDevices` struct. I think your correct, we can remove this var.

3. I was thinking something simpler. When a new Image is created then all devices that use the old image get this bool set to `true`. We could do this async, but it would be more of an implementation detail and not really affect the flow.

**Review by acosferreira — APPROVED** [review]
lgtm

**Pull request merged**